application/src/main/java/org/togetherjava/tjbot/commands/help/AskCommand.java

```diff
@@ -77,6 +86,30 @@ public AskCommand(Config config, HelpSystemHelper helper) {
 
     @Override
     public void onSlashCommand(SlashCommandInteractionEvent event) {
+        if (isUserOnCooldown(event.getUser())) {
+            String message =
+                    """
+                            You can only create 1 help thread per 5 minutes. While you are waiting changing the infos of your previous thread may help.
+                            Use these commands in the help thread to change its details:-
+                            %s
+                            %s""";
```

Comment:
* It is crucial that we link the existing thread
* Needs some friendlyness and grammar rework.

> Sorry, you can only create a single help thread every 5 minutes. Please use your existing thread #foo instead.
> If you made a typo or similar, you can adjust the title using the command /foo and the category with /foo 👌

Reply:
done